.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+minimum_pre_commit_version: "2.9.0"
+repos:
+  - repo: "https://github.com/psf/black"
+    rev: "21.4b2"
```

Comment:
Can't we just omit the `rev` to always get the latest version?

Reply:
the same as before